lxd/storage/drivers/driver_powerflex.go

```diff
@@ -0,0 +1,208 @@
+package drivers
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/canonical/lxd/lxd/migration"
+	"github.com/canonical/lxd/lxd/operations"
+	"github.com/canonical/lxd/shared"
+	"github.com/canonical/lxd/shared/api"
+	"github.com/canonical/lxd/shared/validate"
+)
+
+// powerFlexDefaultUser represents the default PowerFlex user name.
+const powerFlexDefaultUser = "admin"
+
+// powerFlexDefaultSize represents the default PowerFlex volume size.
+const powerFlexDefaultSize = "8GiB"
+
+var powerFlexLoaded bool
+var powerFlexVersion string
+
+type powerflex struct {
+	common
+
+	client *powerFlexClient
+}
+
+// load is used to run one-time action per-driver rather than per-pool.
+func (d *powerflex) load() error {
+	// Done if previously loaded.
+	if powerFlexLoaded {
+		return nil
+	}
+
+	// Detect and record the version.
+	out, err := shared.RunCommand("nvme", "version")
+	if err != nil {
+		return err
```

Comment:
Lets wrap this error to help users understand what it means

Reply:
Wrapped.